internal/neofs/neofs.go

```diff
@@ -113,44 +109,10 @@ func (x *NeoFS) Container(ctx context.Context, idCnr cid.ID) (*container.Contain
 	return res, nil
 }
 
-// ContainerExists implements authmate.NeoFS interface method.
-func (x *NeoFS) ContainerExists(ctx context.Context, idCnr cid.ID) error {
-	_, err := x.pool.GetContainer(ctx, &idCnr)
-	if err != nil {
-		return fmt.Errorf("get container via connection pool: %w", err)
-	}
-
-	return nil
-}
-
-// PrmContainerCreate groups parameters of CreateContainer operation.
-type PrmContainerCreate struct {
-	// NeoFS identifier of the container creator.
-	Creator owner.ID
-
-	// Container placement policy.
-	Policy netmap.PlacementPolicy
-
-	// Name for the container.
-	Name string
-
-	// Time when container is created.
-	Time time.Time
-
-	// Basic ACL of the container.
-	BasicACL acl.BasicACL
-
-	// Token of the container's creation session (optional, nil means session absence).
-	SessionToken *session.Token
-
-	// Attributes for optional parameters.
-	AdditionalAttributes [][2]string
-}
-
 // CreateContainer constructs new container from the parameters and saves it in NeoFS
 // using connection pool. Returns any error encountered which prevent the container
 // to be saved.
-func (x *NeoFS) CreateContainer(ctx context.Context, prm PrmContainerCreate) (*cid.ID, error) {
+func (x *NeoFS) CreateContainer(ctx context.Context, prm neofs.PrmContainerCreate) (*cid.ID, error) {
```

Comment:
If method just follows some interface declaration from now, lets change its docs to just
```
// CreateContainer implements neofs.NeoFS interface method.
```
Same for all other methods.